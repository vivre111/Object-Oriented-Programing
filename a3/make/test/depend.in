abc: abc1
abc: abc2
abc: abc3
abc: abc4
abc: abc5
abc: abc6
abc: abc7
abc: abc8
abc: abc9
abc: abc10
abc: abc11
abc: abc12
abd: abd1
abd: abc1
abd: abd1
abd: abd2
abd: abc
abc1: abc12
abc1: abc13
abc1: abc14
abc1: abc15
abc1: abc16
abc1: abc17
abc1: abc18
abc1: abc19


