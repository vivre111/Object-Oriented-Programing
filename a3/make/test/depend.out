Max dependencies exceeded
Max dependencies exceeded
Max targets exceeded
Max targets exceeded
Max targets exceeded
